### DISCUSSION TITLE
有没有user-rule.txt的使用demo吗？

### DISCUSSION THREAD

**Discussion by zZZ-github-life** [shadowsocks/shadowsocks-windows#3426] [Q&A]
配置user-rule.txt这个文件，和pac.txt一样吗，有没有示例

**Comment by chenshaoju** [comment]
可以参考： https://adblockplus.org/en/filter-cheatsheet

**Reply by zZZ-github-life** [reply]
直接在user-rule.txt 文件里，每写一个规则换行就行了吗？ 每个规则需要用双引号吗，或者写成数组？

**Reply by chenshaoju** [reply]
比如说：
![image](https://github.com/shadowsocks/shadowsocks-windows/assets/515763/a677c35b-7066-457e-97b6-c821168c6149)

**Reply by zZZ-github-life** [reply]
好的，感谢！
